data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/DefaultPluginMapper.java

```diff
@@ -0,0 +1,47 @@
+package org.opensearch.dataprepper.logstash.mapping;
+
+import com.amazon.dataprepper.model.configuration.PluginModel;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.opensearch.dataprepper.logstash.exception.LogstashMappingException;
+import org.opensearch.dataprepper.logstash.model.LogstashPlugin;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Converts Logstash plugin model to Data Prepper plugin model using mapping file
+ *
+ * @since 1.2
+ */
+public class DefaultPluginMapper implements LogstashPluginMapper {
+    private final ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+
+    @Override
+    public PluginModel mapPlugin(LogstashPlugin logstashPlugin) throws IOException {
+
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(logstashPlugin.getPluginName() + ".mapping.yaml");
```

Comment:
Please extract a variable for this: `logstashPlugin.getPluginName() + ".mapping.yaml"`.

You use it below.

And another comment suggests another place to use it.

Reply:
I would like to get a uniform naming convention on this. 
Are we using extension `.yml` or `.yaml` or supporting both?
In our configuration [README](https://github.com/opensearch-project/data-prepper/blob/main/docs/getting_started.md#configure-data-prepper), we mention `.yaml` but in our [examples](https://github.com/opensearch-project/data-prepper/tree/main/examples) we use `.yml`

Reply:
I think we should use `.yaml` consistently.

It is also FAQ 1 out of 2.

https://yaml.org/faq.html